pkgs/applications/graphics/monado/default.nix

```diff
@@ -46,25 +46,16 @@
 
 stdenv.mkDerivation rec {
   pname = "monado";
-  version = "21.0.0";
+  version = "21.0.0+date=2022-05-21";
 
   src = fetchFromGitLab {
     domain = "gitlab.freedesktop.org";
     owner = pname;
     repo = pname;
-    rev = "v${version}";
-    sha256 = "07zxs96i3prjqww1f68496cl2xxqaidx32lpfyy0pn5am4c297zc";
+    rev = "490e90b4938408f357ff2781dbf93841ee9e4f6a";
+    sha256 = "sha256-sHMI7yHmLiMHMgCOUHDWGSLRXe52tEj17N6Z3l92vPI=";
   };
 
-  patches = [
-    # https://github.com/NixOS/nixpkgs/issues/137245
-    # Fix warning after Vulkan 1.2.174 VK_NULL_HANDLE change
-    (fetchpatch {
-      url = "https://gitlab.freedesktop.org/monado/monado/-/commit/c47775a95d8e139a2f234063793eb6726f830510.patch";
-      sha256 = "093ymvi9ifpk4vyjcwhhci9cnscxwbv5f80xdbppcqa0j92nmkmp";
-    })
-  ];
-
```

Comment:
Since removing the patches, `fetchPatch` can be removed from above

Reply:
Done